pom.xml

```diff
@@ -247,6 +251,11 @@
             <version>${confluent.log4j.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.core</groupId>
+            <artifactId>jersey-common</artifactId>
```

Comment:
let's skip jersey which we'll decide later whether to upgrade to 2.34 or 3.0.2 and let's change the title to mention the important fix on mapper asl which will fix a breaking change in newer branches